Gemfile

```diff
@@ -14,6 +14,7 @@ gem "sprockets-rails", "~> 3.4"
 gem "stimulus-rails", "~> 0.7"
 gem "turbo-rails", "~> 1.0"
 gem "view_component", "~> 2.69"
+gem "lookbook", "~> 0.9.6"
```

Comment:
Lookbook v1.0.0 was released _today_! Let's make sure we use the latest since this is now stable.

https://github.com/allmarkedup/lookbook/releases/tag/v1.0.0

Reply:
Updated!